lib/wanda_web/auth/user_detector.ex

```diff
@@ -0,0 +1,19 @@
+defmodule WandaWeb.Auth.UserDetector do
+  @moduledoc """
+  This module allows extracting a very simple representation of a user from a connection.
+
+  Only the user_id and abilities are available.
+  """
+
+  alias Wanda.Users.User
+
+  def current_user(%{private: %{user_id: user_id, abilities: abilities}})
+      when not is_nil(user_id) and is_list(abilities) do
```

Comment:
There is no way that `abilities` is something that is not a list, right?

Reply:
Hopefully not 😄 being extra careful. Do you prefer us to remove it?

Reply:
Actually if in the jwt claims we have `"abilities" => null`, that might be a problem. Very edgy tho

Reply:
I personally would. Now... Up to you :stuck_out_tongue_closed_eyes:

Reply:
> Actually if in the jwt claims we have `"abilities" => null`, that might be a problem. Very edgy tho

But you are converting that to an empty list in the jwt code conversion